### DISCUSSION TITLE
Working with a friend to make a scene in spoke

### DISCUSSION THREAD

**Discussion by shunphoenix55** [mozilla/hubs#4212] [Spoke Questions] [answered]
Is there a way to work together with someone to create a scene in spoke over the internet? Kinda like how Unity does it.

**Comment by j-conrad** [comment, marked as answer]
As of this answer, there is not. Each Hubs/Spoke account is tied to an individual. 
As a workaround, some people will make their scene 'Remixable' when they publish it so it's easier for someone else to make a new version, edit it, and publish again. You can also go to Spoke's file menu and choose '_Export as legacy .spoke project_', then send someone the project for them to import and change.
